pkg/frontend/admin_openshiftcluster_kubernetesobjects_get.go

```diff
@@ -50,9 +51,9 @@ func (f *frontend) _getAdminKubernetesObjects(ctx context.Context, r *http.Reque
 	}
 
 	if name != "" {
-		return f.kubeActions.Get(ctx, doc.OpenShiftCluster, kind, namespace, name)
+		return ka.Get(ctx, doc.OpenShiftCluster, kind, namespace, name)
```

Comment:
Are the unittests working? What is your plan with mocking?

Reply:
They aren't at the moment. I will work on that.